packages/components/src/toggle-group-control/stories/index.js

```diff
@@ -101,6 +106,16 @@ Default.args = {
 	],
 };
 
+export const WithTooltipText = _default.bind( {} );
+WithTooltipText.args = {
+	...Default.args,
+	options: [
+		{ value: 1, label: '1', tooltipText: 'One' },
+		{ value: 2, label: '2', tooltipText: 'Two' },
+		{ value: 3, label: '3', tooltipText: 'Three' },
+	],
+};
```

Comment:
This will need updating to use `aria-label` via `showTooltip` as well.

```suggestion
export const WithTooltip = _default.bind( {} );
WithTooltip.args = {
	...Default.args,
	options: [
		{ value: 1, label: '1', showTooltip: true, 'aria-label': 'One' },
		{ value: 2, label: '2', showTooltip: true, 'aria-label': 'Two' },
		{ value: 3, label: '3', showTooltip: true, 'aria-label': 'Three' },
	],
};
```